### ISSUE TITLE
Node temperature constant when plant loop is off even with embedded pipes

### ISSUE THREAD

**Issue by jmarrec** [NREL/EnergyPlus#6244]
Issue overview
--------------

I'm not sure I'll be able to phrase this issue correctly the first time around but I'll try. I'm using a `ZoneHVAC:LowTemperatureRadiant:VariableFlow` object that is embedded in the wall of a **large** trombe wall zone (around 1000 meter of embedded piping!). I'm actually trying to use this as a heat source to heat up a water tank connected to the supply side of a condenser loop (Use Side of the WaterHeater:Mixed). The Water heater's source side is on the supply side of another loop which has the LowTempVarFlow object on the demand side. I was trying to use an availabilityManager:DifferentialThermostat to turn ON the loop when the LowTemp outlet node temperature is greater than the tank temperature + 5°C.

Turns out this doesn't work because when the loop is off, the node temperature stays constant. I see my trombe loop work until the end of March, and then it stays off constantly until the end of year: the LowTemp Outlet Node temperature stays constant and too high for the AVM:DifferentialThermostat to turn the loop back on again. And there are plenty of times where if I consider the Trombe Zone Mean Radiant Temperature and the Water Tank temperature, useful heating could have occured.

I understand that in most cases loop distribution losses are ignored (maybe not if you use a one of the non Adiabatic Pipes), but in this case I think it's a problem that it doesn't consider what's happening within the wall even when the loop is off. The loop temperature should be updated.

What do you think?

### Details

Some additional details for this issue (if relevant):
 - Platform: All
 - Version of EnergyPlus: tested on 8.5 and briefly on 8.7

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
Interesting use of the `ZoneHVAC:LowTemperatureRadiant:VariableFlow` object. Another limitation to consider here - I am not sure if the thermal mass of the water in the embedded pipes is accounted for.  For your immediate need, would it work to use `ZoneHVAC:LowTemperatureRadiant:ConstantFlow` with a zero pump power just to force the water temperature in the slab to update?  @RKStrand any thoughts here?

**Comment by jmarrec** [comment]
I would be fine using a low temp constant flow, I'm actually not using the variable capacity at the moment.
I'd be fine running a pump constantly but can I make sure I bypass the waterheater:mixed (no heating element, just a tank) unless I have a big enough delta T? I wouldn't want to make the tank colder, that would defeat the purpose.

(In the meantime I've set up an EMS program that senses for zone mean radiant temperature - perhaps I should use a sensor on the surface itself? - that mimic the behavior of the avm:differential thermostat, I think it may be working...)

**Comment by RKStrand** [comment]
Kudos on giving this a try and I agree with @mjwitte that this is an interesting use of the variable flow radiant system.  What you really have is a radiant "cooling" system in that you are cooling your Trombe wall zone by heating up your water.  Are you treating this as a cooling system and using cooling controls?  I don't have experience with the EMS in EnergyPlus so I can't comment on that.  Basically, to get the water temperature to update you have to somehow get the system itself to operate, but just getting the temperature to update is only part of the issue as you still have to have flow through the system to get it to calculate what the result temperature coming out of the system will be.  The water temperature really doesn't have much meaning if the system isn't circulating water.

To answer one point that was brought up above, the thermal mass of the water in the pipes is not accounted for in the model.

**Comment by jmarrec** [comment]
Correct, I'm using treating it as a cooling loop, using a `PlantEquipmentOperation:CoolingLoad`,
 with the `WaterHeater:Mixed `listed there, a SPM placed on the plant supply outlet of 5C, and the Low Temp Radiant is also controlled to MRT=5°C.

**Comment by jmarrec** [comment]
I've sent a request to the helpdesk with the file, the EMS program I implemented doesn't work, in the sense that the EMS program does work (the actuators - Low Temp flowrate and Loop pump flow rates - are set correctly), but I get cases where the supply side of the plant loop has a given mass flow rate, the demand side has no flow (none through the lowtemp neither the bypass branch). There's no common pipe simulation in this plant loop, so I don't think this should be happening...

If anyone can offer insight that'd be very very much appreciated
